drivers/amazonec2/amazonec2.go

```diff
@@ -327,8 +332,8 @@ func (d *Driver) GetIP() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	d.IPAddress = inst.IpAddress
-	return d.IPAddress, nil
+
```

Comment:
+1 if this is called from `docker-machine ip` it does not even save the config so it doesn't matter to set this field in the struct